GoogleForCreators/web-stories-wp
# Video Optimization: Improve handling of files exceeding upload limit

## Context



## Summary

Scenario: the site's upload limit is 100MB. You upload a file that is 150MB large.

Original size: 170MB (mp4) Transcode size: 6 MB

## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions

1. Download a large video. [Example](https://www.pexels.com/video/different-kinds-of-vehicles-on-the-freeway-2053100/).
2. Create a new story.
3. Drag video into canvas. 
4. Video should transcode and upload. 

### QA






This PR can be tested by following these steps:

1.

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #6580
